README.md

```diff
@@ -133,23 +133,59 @@ docker-compose -f docker-compose.test.yml up --build --exit-code-from test
 ~~~
 
 ### Developing / Debugging
-Build and start your container in "developer mode"
+
+1.  Build your container in "developer mode":
 ~~~bash
-./scripts/devel-compose build --no-cache vmaas_webapp
-./scripts/devel-compose up vmaas_webapp
+./scripts/devel-compose build --no-cache
+./scripts/devel-compose up
 ~~~
 
-switch inside of the container
+2.  Switch into **webapp** container:
+
 ~~~bash
-./scripts/devel-compose exec vmaas_webapp bash
+docker-compose exec vmaas_webapp bash
 ~~~
 
-now your local git directory is mounted under `/git` in the container so any change
+Now your local git directory is mounted under `/git` in the container so any change
 you make you can immediatelly test.
 
-```[root@4cb6b50d0cb6 git]# python3 ./app.py```
+```[root@localhost git]# python3 ./app.py```
 
-Note that by default container does NOT run the application (so you can run your own modificationtion)
+Note that by default container does NOT run the application 
+(so you can run your own modificationtion)
 so if you want to run "original" (unmodified) application use
 
-```[root@4cb6b50d0cb6 git]# /app/entrypoint.sh```
+```[root@localhost git]# /app/entrypoint.sh```
+
+Now you have bash inside vmaas_webapp container, run: 
+
+```[root@localhost git]# ./entrypoint.sh```
+
+3.  Switch into **websocket** container:
+~~~bash
+docker-compose exec vmaas_websocket bash
+~~~
+
+Now run:
+
+```[root@localhost git]# python3 websocket.py```
+
+4.  Switch into **reposcan** container:
+~~~bash
+docker-compose exec vmaas_reposcan bash
+~~~
+
+Now run:
+
+
+```[root@localhost git]# ./entrypoint.sh```
+
+5.    Switch to **database** container to gain access to db:
+ 
+~~~bash
+docker-compose exec vmaas_database bash
+~~~
+
+Now you can run database terminal with:
+
+```[root@localhost git]# psql -d vmaas```
```

Comment:
You have many extra spaces and empty lines here. It's not necessary. Could you remove it, please?